{"name": "test_trigger_test_button", "status": "failed", "statusDetails": {"message": "AssertionError: Failed\nassert 1 == 2", "trace": "def test_trigger_test_button():\n        print(\"Testing Trigger Test Button\")\n    \n>       assert 1==2, \"Failed\"\nE       AssertionError: Failed\nE       assert 1 == 2\n\ntests\\test_stream.py:6: AssertionError"}, "attachments": [{"name": "stdout", "source": "507c41c7-b54c-4019-a53c-7ca07950f39e-attachment.txt", "type": "text/plain"}], "start": 1740428633708, "stop": 1740428633708, "uuid": "acbe3830-2bcd-4a12-84f2-ccba6126ffc2", "historyId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "testCaseId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "fullName": "tests.test_stream#test_trigger_test_button", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_stream"}, {"name": "host", "value": "DESKTOP-LUCDFB2"}, {"name": "thread", "value": "4960-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_stream"}]}